src/Shell.ts

```diff
@@ -30,12 +30,12 @@ export default class Shell {
   }
 
   /**
-   * Executes a command in child process and returns it.
+   * Executes a command synchronously.
    * It must be used when we need to utilize shell functionality such as pipe, redirects.
    * @param {string} command Command string to execute.
-   * @returns Child process that was spawned by this call.
+   * @returns Buffer object.
    */
-  public static executeInShell(command: string): ChildProcess {
-    return exec(command);
+  public static executeSyncInShell(command: string): Buffer {
```

Comment:
I would leave the command name `executeInShell` without the Sync.

We don't have any async commands after #40, which changes also `spawn` above to `spawnSync`:
https://github.com/OpenST/mosaic-chains/pull/40/files#diff-72c4182311d665d1eaa110bfff23e75fR29